web/packages/teleterm/src/ui/DocumentCluster/DocumentCluster.tsx

```diff
@@ -142,10 +142,5 @@ const Layout = styled(Box)`
   flex-direction: column;
   display: flex;
   flex: 1;
-  max-width: 1248px;
-
-  ::after {
-    content: ' ';
-    padding-bottom: 24px;
-  }
+  height: max-content;
```

Comment:
I was going to suggest explaining in a comment why this is needed. But then I thought "If it's needed here, why do we not have this for every other component?". Then I realized that it's necessary only because `UnifiedResources` uses `FeatureBox` which sets `height: 100%` and then `UnifiedResources` also sets `margin: auto` on that `FeatureBox`.

If margin was set to `0 auto` and height wasn't set at all (as in used the default `auto` value) then we wouldn't need to use max-content.

Maybe we could make `UnifiedResources` accept `height` prop so that we can set it to `auto`? TBH, Connect doesn't need `FeatureBox` at all, so perhaps this should be moved one level up or something (that might be tricky because of that `ContainerContext` class). It's an import from `teleport` anyway so at some point we'll probably have to remove it either way.

I'd rather avoid those `height: max-content` workarounds because they seem like code smell. We already handle overflow on a global level (per `<Document>` component) so all that a component needs to do is simply grow its content. `FeatureBox` is meant for another context and it looks like we shouldn't use it within Connect.

Reply:
@avatus Do you remember why #30988 uses `margin: auto` on `FeatureBox` instead of `margin: 0 auto`?

Reply:
I don't exactly remember why. Could have been vestigial from previous testing trying to get the component to center within the container context but I don't remember

Reply:
>I'd rather avoid those height: max-content workarounds because they seem like code smell. We already handle overflow on a global level (per <Document> component) so all that a component needs to do is simply grow its content. FeatureBox is meant for another context and it looks like we shouldn't use it within Connect.

Agree. I have to admit that I couldn't localize the issue that was causing the `UnifiedResource` component to not grow properly in Connect and decided to use `height: max-content`.

I will fix it and remove `FeatureBox` from the shared component. I will do this in another PR which will be backported to v14.